docker/1.6.0/py2/Dockerfile.gpu

```diff
@@ -18,7 +18,7 @@ ENV LANG=C.UTF-8 \
 
 RUN apt-get update \
  && apt-get install -y --no-install-recommends \
-    # build-essential needed to pip install sagemaker_mxnet_container
+    # build-essential needed to pip install sagemaker_mxnet_training_toolkit
     build-essential \
```

Comment:
Can build-essential be removed from the dockerfile once sagemaker_mxnet_training_toolkit can be installed from pypi?

Reply:
as far as i remember it's not really necessary for sagemaker_mxnet_container but is required so that customers could install packages from pypi as part of their user scripts running in sagemaker.

Reply:
Oh, I thought that both build-essential and python-dev were only needed in installing packages from source.

Thanks for clarifying.